server/database/queries/getProfile/index.js

```diff
@@ -0,0 +1,10 @@
+const connection = require('../../config/connection');
+
+const getProfile = (id) =>
+  connection.query({
+    text:
+      'select image, firstname, lastname, email, weight, height, gender, age  from users WHERE id = $1;',
```

Comment:
You might be able to change this query to `select * from users where id = $1;` instead of listing each field (unless there's a field you want to hide).  Then, in the future if you add more columns to that table, your client will receive them automatically without you having to remember to change this query.

Reply:
When selecting all, the password is returned. It is necessary not to return it for security reasons

Reply:
Haha, yes, that'd be bad.  This is fine then!